entrypoint.sh

```diff
@@ -11,14 +11,11 @@ echo Running makemigrations
 echo Running migrations
 ./manage.py migrate
 
-echo "Ensuring our base configuration is present in the database"
+echo Ensuring our base configuration is present in the database
```

Comment:
👎 on this change since it means the string will be highlighted following shell syntax rules in, for example, GitHub diffs

Reply:
I wasn't sure which way was preferred, I made it consistent with the most recent changes. I'll put it back